eng/pipelines/pr.yml

```diff
@@ -9,6 +9,7 @@ jobs:
 
     strategy:
       matrix:
```

Comment:
ok so we are getting an issue that was due to happen, pnpm version we are using is broken with node 20. 
But if we update the version(in rush.json) then rush needs to update to using pnpm workspaces with the latest pnpm.